src/main/resources/peer-review-transform.xsl

```diff
@@ -103,41 +99,35 @@
               <xsl:value-of select=".//named-content[@content-type = 'letter-date']" />
             </time>
             <div class="letter__title">
-              <!-- trigger for expand and collapse -->
-              <a class="letter--toggle" data-toggle="collapse" href="#">[Acceptance Letter]</a>
-              <!-- end trigger for expand and collapse -->
+              <a class="peer-review-accordion-expander" href="#">[Acceptance Letter]</a>
             </div>
             <!-- accordion container -->
-            <div itemprop="reviewBody" class="collapse" id="decisionLetter3">
+            <div itemprop="reviewBody" class="peer-review-accordion-content">
               <div class="letter__body">
                   <xsl:apply-templates select="body" />
               </div>
             </div>
-            <!-- end accordion container -->
           </div>
         </div>
       </td>
     </tr>
   </xsl:template>
 
   <xsl:template match="sub-article[@article-type = 'author-comment']">
-    <tr>
+    <tr class="peer-review-accordion-item">
       <td class="letter">
         <div class="author-response">
-          <!-- author response -->
           <time class="letter__date">
             <xsl:value-of select=".//named-content[@content-type = 'author-response-date']" />
           </time>
           <div class="letter__title">
-            <a class="letter--toggle" data-toggle="collapse" href="#decisionLetter2">[Author Response]</a>
+            <a class="peer-review-accordion-expander" href="#">[Author Response]</a>
```

Comment:
Can you change 
`[Author Response]` to `Author Response`
(and remove from the other toggles)

Reply:
will do, thanks!